### PR TITLE
Redirect Chargebee to Apps

### PR DESCRIPTION
**Pull request by fjvallarino**
Since ```STORE_URL``` is redefined in the Store and its value replaced with "", I created a copy named ```STORE_URL_CH```. I use its value to check if the user is currently in Apps or Store, and what the current path is (I could use routes for that part). Let me know if you have a better idea on how to implement this (or a better name for the new constant).

@Rise-Vision/apps please review

### CODE REVIEW

File: src/js/dtv-common-header.js [hunk 1]
```diff
@@ -167,6 +166,17 @@ angular.module("risevision.common.header", [
         event.preventDefault();
       }
     });
+
+    $rootScope.$on("risevision.company.selectedCompanyChanged", function () {
+      // Redirect Store requests by Chargebee users to Apps
+      var currentURL = $window.location.href;
+      var isStoreRequest = currentURL.indexOf(STORE_URL_CH) >= 0;
+      var isInvoicesRequest = currentURL.indexOf(STORE_URL_CH + "account/view/invoicesHistory") >= 0;
```
Comment (alex-deaconu) [inline src/js/dtv-common-header.js]: @fjvallarino I don't follow why this check is in the Common-Header? We should move this to the Store as it specific to that App. Thanks.

**Comment by fjvallarino** [comment]
@alex-deaconu moved redirection logic to Store and removed unneeded constants.

**Comment by alex-deaconu** [comment]
@fjvallarino looks good 👍 

Just curious re: unit test changes, does that fix the randomly failing unit test we've seen before?

Thanks!

**Comment by fjvallarino** [comment]
@alex-deaconu the useFakeTimers thing fixes this issue:

image

**Comment by alex-deaconu** [comment]
Nice!!

**Comment by olegrise** [comment]
👍

**Pull request merged**